package.json

```diff
@@ -44,7 +45,11 @@
     "android": "(cd ./native && npm run android)",
     "check-owners": "npm whoami && grep -E $(npm whoami) NPM_OWNERS && node scripts/npm/check-owners.js",
     "publish": "npm run check-owners && npm run build && git pull --rebase && npm run test && lerna publish",
-    "danger": "danger"
+    "danger": "danger",
+    "prettier": "prettier --config .prettierrc --write \"{packages,native,scripts}/**/*.js\" dangerfile.js webpack.config.babel.js"
```

Comment:
Surely `prettier --config .prettierrc --write \"**/*.js\"` is enough here?

Reply:
Yeah you are probably right